packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -126,10 +129,19 @@ class ChildSceneLayer extends Layer {
     this.sceneToken
   });
 
+  /// Offset from parent in the parent's coordinate system.
   Offset offset;
+
+  /// The number of physical pixels the child should produce for each logical pixel.
```

Comment:
Is this really physical pixels? e.g. consider if the ChildSceneLayer is inside a transformed layer with a scale and a rotation and a skew.